[Rust] use structs to model data instead of Document in CRUD fundamentals (#95)
mongodb/docs-rust: source/fundamentals/crud/read-operations/query.txt

      :language: json
      :visible: false

      Fruit { _id: "2", name: "apple", quantity: 4, description: Some("Granny Smith"),
      vendors: None }
      
.. note::

>    The :ref:`struct describing the documents<rust-query-sample-data>` uses ``#[serde(skip_serializing_if =
>    "Option::is_none")]`` field attributes to specify that the data be ignored if it is
>    specified as ``None``. This prevents a description field of ``None`` from being returned on am ``$exists`` query.

Reviewer: S: I don't know Rust very well, but this note was hard for me to follow. My best attempt at clarifying would look like this:

```suggestion
   The :ref:`Fruit struct describing the documents in this guide <rust-query-sample-data>` uses the ``#[serde(skip_serializing_if =
   "Option::is_none")]`` attribute on two of its fields. This attribute specifies that the field be ignored if its
   value is ``None``. This prevents a ``description`` value of ``None`` from being returned on an ``$exists`` query.
```
